src/components/boilers/Boiler.jsx

```diff
@@ -4,9 +4,12 @@ import { FaTimes as DeleteIcon } from 'react-icons/fa';
 import { MdEdit as EditIcon } from 'react-icons/md';
 import styles from './Boiler.module.css';
 import { getBuilding } from '../../store/buildings';
+import { useDispatch } from 'react-redux';
+import { deleteBoiler } from '../../redux/actions/boilersActions';
 
-export const Boiler = ({ boiler, onDelete, onModify }) => {
+export const Boiler = ({ boiler, onModify }) => {
   const { id, type, isInstalled, maintenanceTimeMinutes, buildingId } = boiler;
+  const dispatch = useDispatch();
   const building = getBuilding(buildingId);
```

Comment:
This way of getting the building is outdated, you should now use the useSelector hook :)